aip/general/0140.md

```diff
@@ -88,6 +88,31 @@ For uniformity, field names that contain both a noun and an adjective
 - `collected_items` (**not** `items_collected`)
 - `imported_objects` (**not** `objects_imported`)
 
+### Verbs
+
+Field names **must not** be expressed to reflect an intent or action.
+That is, they **must not** be expressed as verbs. Rather, the field name
+should express the condition of that aspect of the resource as defined
+by the value of the field in question.
+
+When performing a mutation, the field value is set to the desired
+*final* value of that property. When performing a read, the field value
+is returned as the *current* value of that property. As such, the name
+should always be *a noun or an adjective* as it describes what is so,
+not what to do.
+
+- `collected_items` (**not** `collect_items`)
+- `imported_objects` (**not** `import_objects`)
+
+Alternatively, method names, whether standard or custom, express intent
+to change the condition of resources and **should** be named as verbs,
```

Comment:
I think we should avoid guidance on Methods in this Field-specific AIP. As such I would reword this as a statement of fact.
```suggestion
to change the condition of resources and are named as verbs,
```